test/c_api_tests.cpp

```diff
@@ -94,6 +94,43 @@ TEST(CAPITests, TokenizerCAPI) {
 #endif
 }
 
+TEST(CAPITests, ChatTemplate) {
+#if TEST_PHI2
+  auto config = OgaConfig::Create(PHI2_PATH);  // just to create tokenizer (phi-2 does not have a chat template)
+  auto model = OgaModel::Create(*config);
+  auto tokenizer = OgaTokenizer::Create(*model);
+
+  // Testing phi-4 chat template
+  {
+    std::string messages_json = R"(
```

Comment:
Why is this a std::string if you're just doing c_str() on it? chat_template is a regular C string right after it. Same with expected_output

Reply:
good point, changed it to a regular C string